src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js

```diff
@@ -397,6 +401,16 @@ YmailApi.StormUI.prototype.displayFailure = function(message, opt_type) {
 };
 
 
+/**
+ * Log an interesting event
+ * @param {string} name
+ * @param {*} tags
```

Comment:
The tags cannot have indeterministic values, i.e. the possibilities should be finite.